package.json

```diff
@@ -29,7 +29,7 @@
     "@oclif/config": "^1.17.0",
     "@salesforce/command": "^3.1.0",
     "@salesforce/core": "^2.18.0",
-    "@salesforce/lwc-dev-mobile-core": "^1.0.0-beta9",
+    "@salesforce/lwc-dev-mobile-core": "file:../lwc-dev-mobile-core/salesforce-lwc-dev-mobile-core-1.0.0-beta10.tgz",
```

Comment:
Using tarball locally since the core lib is not published yet. So it won't build on CI.

Reply:
Using beta 10 now.